scripts/build.py

```diff
@@ -1317,6 +1317,16 @@ def generate_build_directory(hashes):
     python_utils.PRINT('Build completed.')
 
 
+def generate_python_package():
+    """Generates hashes for files. Minifies files and interpolates paths
+    in HTMLs to include hashes. Renames the files to include hashes and copies
+    them into build directory.
+    """
+    python_utils.PRINT('Building Oppia package...')
+    subprocess.check_call('python setup.py sdist -d build', shell=True)
```

Comment:
Typically don't we use a list of strings here (in the first argument)?

Reply:
Yep, done.